provider/pkg/await/await_test.go

```diff
@@ -528,64 +521,8 @@ func Test_Deletion(t *testing.T) {
 				inputs:  validPodUnstructured,
 				outputs: validPodUnstructured,
 			},
-			awaiter: nil,
-			expect:  []expectF{succeeded(), deleted("default", "foo")},
-		},
-		{
-			name: "WatchTimeout",
-			args: args{
-				resType: tokens.Type("kubernetes:core/v1:Pod"),
-				name:    "foo",
-				objects: []runtime.Object{validPodUnstructured},
-				inputs:  validPodUnstructured,
-				outputs: validPodUnstructured,
-			},
-			reaction: []reactionF{suppressDeletion},
-			awaiter:  nil,
-			watcher:  withWatchClosed(watch.NewRaceFreeFake()),
-			expect:   []expectF{failed(&timeoutError{})},
-		},
-		{
-			name: "WatchTimeoutWithRecovery",
-			args: args{
-				resType: tokens.Type("kubernetes:core/v1:Pod"),
-				name:    "foo",
-				objects: []runtime.Object{validPodUnstructured},
-				inputs:  validPodUnstructured,
-				outputs: validPodUnstructured,
-			},
-			reaction: nil,
-			awaiter:  nil,
-			watcher:  withWatchClosed(watch.NewRaceFreeFake()),
-			expect:   []expectF{succeeded()},
-		},
-		{
-			name: "WatchError",
```

Comment:
These are no longer needed because we're using an Informer which takes care of these errors for us.